packages/driver/src/cy/commands/screenshot.coffee

```diff
@@ -82,6 +82,11 @@ scrollOverrides = (win, doc) ->
     win.scrollTo(originalX, originalY)
 
 takeScrollingScreenshots = (scrolls, win, state, automationOptions) ->
+  if scrolls.length is 0
+    return Promise.reject(
+      new Error("It was not possible to take a screenshot, since the number of scrolls calculated to do so was zero.")
```

Comment:
I think the wording on this message is a bit strange. We try to make our error messages "actionable", so there's an error that the user is encountering, that they likely weren't expecting, what can they do moving forward to fix this error message. This one leaves me confused. 

Is this check in case they pass in a negative padding? Are they screenshotting a zero width element? 

1. What's the actual situation the user has put themselves in that causes this error?
2. What can the user do to get rid of this error?

Reply:
I was fixing a bug in Cypress.

If an element has a height of zero, then [`numScreenshots`](https://github.com/cypress-io/cypress/blob/c0962511f045f07f2dc255b251ac251b2d8ae718/packages/driver/src/cy/commands/screenshot.coffee#L138) ends up being `0`, then `scrolls` ends up with a length of `0`, then [`_.last(results)` produces `undefined`](https://github.com/cypress-io/cypress/blob/c0962511f045f07f2dc255b251ac251b2d8ae718/packages/driver/src/cy/commands/screenshot.coffee#L99), and finally [destructuring from `props` throws a `TypeError`](https://github.com/cypress-io/cypress/blob/c0962511f045f07f2dc255b251ac251b2d8ae718/packages/driver/src/cy/commands/screenshot.coffee#L357).

I agree that it's a difficult to understand error message. But I don't know what you want Cypress to do when somebody tries to take a screenshot of an element with no height? We could make it not error at all, but the saved image then isn't valid.

Reply:
Ah - you already mentioned the bug. Yes, it's a solution to #5149. Should we change the error message to mention that the element has zero height?

Reply:
@jennifer-shehane if you have a suggestion as to how best to word this, we'd be more than happy to apply the suggestion 🙂

Reply:
Yes, if the situation is that this error will throw is that the element has zero height, then I would like the error message to say that. The `number of scrolls calculated to do so was zero` is putting too much weight on the user understanding the implementation details. 

```suggestion
      new Error("It was not possible to take a screenshot, since the height and/or width of the screenshot area was zero.")
```

This would definitely be a better case for showing a 'warning' rather than throwing an error - since it wouldn't necessarily affect the tests functionality. This would be a good use case for application in this open issue: https://github.com/cypress-io/cypress/issues/4917

For now, we will throw since that API doesn't exist.

Reply:
Move these errors to `error_messages` found here: https://github.com/cypress-io/cypress/pull/5078/files#diff-aa529599840c5d5edd94b3f4d4ae631a